_docs_integrate/propose-attributes-to-peer.md

```diff
@@ -147,7 +147,23 @@ If the Recipient does not want to deal with any of the Attributes proposed by th
+The Recipient can accept a ProposeAttributeRequestItem contained in the Request for proposing Attributes using an appropriate [AcceptProposeAttributeRequestItemParameter]({% link _docs_integrate/data-model-overview.md %}#acceptproposeattributerequestitemparameters). Depending on whether the Recipient confirms the fittingness of the associated proposed Attribute, it can agree to its creation and send it back to the Sender unchanged or correct the Attribute Value beforehand. To do this, the Recipient needs to specify the proposed Attribute or an Attribute that differs from the proposed Attribute only by the Attribute Value as the value for the `attribute` parameter of the AcceptProposeAttributeRequestItemParameters.
+
+If the Recipient does not want to create a new Attribute, it can alternatively send an existing Attribute back to the Sender. The `attributeId` parameter of the AcceptProposeAttributeRequestItemParameters must be used for that. The existing Attribute may only differ from the proposed Attribute by its Attribute Value. The acceptance of a ProposeAttributeRequestItem leads to the creation of a corresponding [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) with a [LocalAttributeShareInfo]({% link _docs_integrate/data-model-overview.md %}#localattributeshareinfo) contained within its `shareInfo` property. The `content` of the LocalAttribute is the Attribute that the Recipient wants to send back to the Sender. Depending on whether the Recipient confirms the fittingness of the proposed Attribute, it is therefore the proposed Attribute itself or a corrected version of it.
+
+If the Recipient accepted with a new Attribute or an existing one that wasn't shared with the peer before, an appropriate AcceptResponseItem of type [ProposeAttributeAcceptResponseItem]({% link _docs_integrate/data-model-overview.md %}#proposeattributeacceptresponseitem) is generated.
```

Comment:
Also emphazise that no predecessor of the existing Attribute was shared with the peer when a ProposeAttributeAcceptResponseItem is generated.